CMakeLists.txt

```diff
@@ -73,6 +73,7 @@ include(CMakePackageConfigHelpers)
 include(InstallFreeRDPMan)
 include(GetGitRevisionDescription)
 include(SetFreeRDPCMakeInstallDir)
+include(FindCairo)
```

Comment:
Should be in `libfreerdp/CMakeLists.txt` where the `find_package` is

Reply:
Fixed, let me know if you prefere a new PR